examples/graphql-api/app/components/CharacterList.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import { Link } from 'remix';
+import { Characters } from '~/generated/types';
+
+export interface CharacterListProps {
+  data: Characters['results'];
+}
+
+/**
+ * @name CharacterList
+ * @description This component loops over an array of characters and renders
+ * a link to each characters detail page.
+ */
+export const CharacterList: React.FC<CharacterListProps> = (props) => {
```

Comment:
Similar questions as styles, this component is only used in the route, why not keep it there? It's also simple enough to directly inline the JSX in the route where it's being used and it will make the example simpler to read by using less files